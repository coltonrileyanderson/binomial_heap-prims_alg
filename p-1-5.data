10 13 493 ; 22 3 158 ; 17 31 42 ; 26 6 342 ; 4 1 437 ; 
28 12 7 ; 23 18 452 ; 18 7 4 ; 17 30 351 ; 5 21 60 ; 
18 18 231 ; 21 12 105 ; 21 7 99 ; 27 5 20 ; 29 6 435 ; 
9 17 471 ; 28 1 304 ; 3 21 367 ; 1 13 165 ; 22 9 98 ; 
8 24 384 ; 21 17 90 ; 28 0 44 ; 1 1 141 ; 7 28 348 ; 
25 27 125 ; 26 23 223 ; 16 11 203 ; 29 16 130 ; 6 7 39 ; 
31 3 114 ; 16 17 402 ; 17 21 359 ; 18 14 231 ; 14 3 360 ; 
9 20 332 ; 1 15 209 ; 12 6 239 ; 28 0 427 ; 3 28 180 ; 
6 14 207 ; 24 8 90 ; 13 24 286 ; 28 3 395 ; 31 16 141 ; 
19 4 459 ; 3 17 321 ; 9 24 382 ; 10 16 10 ; 6 4 74 ; 
20 31 322 ; 28 25 340 ; 20 4 220 ; 24 31 154 ; 8 13 486 ; 
12 8 213 ; 29 13 164 ; 21 19 63 ; 5 5 465 ; 10 20 499 ; 
9 6 409 ; 3 30 457 ; 7 15 397 ; 6 13 90 ; 19 7 118 ; 
28 0 342 ; 9 12 94 ; 29 28 435 ; 2 21 104 ; 22 24 382 ; 
28 1 448 ; 27 30 157 ; 10 23 304 ; 23 5 471 ; 30 11 110 ; 
30 25 9 ; 10 19 129 ; 6 10 143 ; 27 23 284 ; 19 9 210 ; 
20 5 349 ; 18 10 373 ; 10 19 476 ; 15 2 230 ; 27 24 437 ; 
20 2 435 ; 8 7 141 ; 18 22 220 ; 9 2 234 ; 18 17 354 ; 
23 22 77 ; 2 23 314 ; 21 20 224 ; 23 26 2 ; 16 24 442 ; 
18 19 309 ; 25 13 297 ; 16 17 1 ; 18 27 208 ; 3 1 225 ; 
25 27 436 ; 17 9 435 ; 5 9 437 ; 31 20 42 ; 23 14 74 ; 
11 3 346 ; 24 12 164 ; 9 30 213 ; 4 2 285 ; 6 15 16 ; 
1 15 263 ; 10 7 433 ; 19 0 342 ; 7 15 23 ; 21 21 354 ; 
24 11 126 ; 5 19 273 ; 3 20 289 ; 5 6 246 ; 20 10 26 ; 
4 17 336 ; 11 18 105 ; 11 4 295 ; 26 27 335 ; 15 18 226 ; 
26 0 264 ; 13 25 404 ; 2 10 149 ; 8 4 296 ; 18 2 109 ; 
3 7 462 ; 22 12 142 ; 26 28 437 ; 22 31 80 ; 8 9 171 ; 
8 9 430 ; 1 2 57 ; 11 31 418 ; 15 0 376 ; 17 5 216 ; 
24 31 306 ; 5 29 410 ; 1 6 312 ; 0 27 354 ; 9 10 412 ; 
19 8 340 ; 21 1 468 ; 20 8 217 ; 20 16 80 ; 26 1 303 ; 
25 3 265 ; 23 9 301 ; 29 5 222 ; 24 28 471 ; 2 8 87 ; 
11 29 297 ; 12 25 68 ; 20 23 101 ; 5 31 101 ; 6 23 14 ; 
9 0 300 ; 19 1 273 ; 24 8 280 ; 20 15 406 ; 29 23 230 ; 
26 8 129 ; 19 12 32 ; 10 5 348 ; 10 18 349 ; 1 20 206 ; 
1 16 44 ; 2 6 465 ; 10 18 232 ; 26 28 361 ; 17 31 205 ; 
25 4 326 ; 5 20 268 ; 10 16 352 ; 28 1 294 ; 17 23 176 ; 
1 19 153 ; 7 25 490 ; 26 17 193 ; 22 2 279 ; 21 8 425 ; 
25 10 366 ; 13 7 455 ; 30 23 56 ; 31 17 326 ; 23 22 15 ; 
10 27 136 ; 3 2 193 ; 21 11 109 ; 23 30 367 ; 31 12 373 ; 
10 10 359 ; 17 2 304 ; 8 30 94 ; 25 5 428 ; 15 8 7 ; 
11 28 186 ; 13 17 361 ; 24 14 496 ; 22 5 177 ; 2 19 117 ; 
13 22 168 ; 15 18 10 ; 13 28 115 ; 18 4 131 ; 26 12 409 ; 
22 10 272 ; 8 19 227 ; 22 0 264 ; 28 1 280 ; 15 26 203 ; 
5 19 457 ; 23 21 262 ; 20 9 425 ; 24 4 151 ; 4 25 321 ; 
15 30 248 ; 2 13 270 ; 2 10 82 ; 4 30 293 ; 30 9 121 ; 
17 23 292 ; 7 17 17 ; 16 27 100 ; 20 18 488 ; 14 27 461 ; 
12 31 25 ; 25 26 220 ; 3 28 203 ; 2 21 377 ; 11 10 217 ; 
2 14 44 ; 20 23 377 ; 15 24 228 ; 1 12 64 ; 28 25 17 ; 
28 11 199 ; 22 11 383 ; 18 14 296 ; 8 20 408 ; 18 29 146 ; 
1 5 65 ; 24 31 476 ; 16 31 140 ; 29 16 131 ; 22 21 452 ; 
1 8 128 ; 13 7 408 ; 27 31 414 ; 15 27 229 ; 13 25 145 ; 
18 3 487 ; 
