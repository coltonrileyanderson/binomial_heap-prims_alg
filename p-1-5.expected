0: 10
1: 16(10)10
2: 1(16)44 20(16)80 29(16)130
3: 2(1)57 12(1)64
4: 9(12)94 14(2)44 19(12)32 25(12)68 28(12)7
5: 0(28)44 21(19)63 30(25)9
6: 5(21)60 23(30)56
7: 6(23)14 22(23)77 27(5)20
8: 4(6)74 7(6)39 13(6)90 15(6)16
9: 8(15)7 17(7)17 18(7)4
10: 11(18)105 26(8)129 31(17)42
11: 3(31)114 24(11)126
weight: 1736
